doczrc.js

```diff
@@ -102,6 +102,72 @@ export default {
       props: {
         bg: "var(--color-grey--lightest)",
       },
+      sidebar: {
+        bg: "var(--color-greyBlue--dark)",
+        navGroup: "var(--color-greyBlue--lighter)",
+        navLink: "var(--color-white)",
+        navLinkActive: "var(--color-white)",
+        tocLink: "var(--color-greyBlue--lighter)",
+        tocLinkActive: "var(--color-white)",
+      },
+    },
+    navigation: {
+      level1: {
+        mb: 3,
+        ml: 0,
+        pl: 0,
+        fontFamily: "fonts.heading",
+        fontSize: 2,
+        fontWeight: 800,
+        textTransform: "uppercase",
+        letterSpacing: "0.02rem",
+        "&::before, &.active::before": {
+          display: "none",
+        },
+      },
+      level2: {
+        position: "relative",
+        pl: 3,
+        ml: 0,
+        fontSize: 2,
+        transition: "background 200ms ease 0s",
+        color: "var(--color-greyBlue--lighter)",
+        "&::before": {
+          content: "''",
+          position: "absolute",
+          top: 2,
+          left: 0,
+          width: 6,
+          height: 6,
+          border: "1px solid var(--color-greyBlue--light)",
+          borderRadius: 100,
```

Comment:
Just to confirm this comes out as `100%`?

Reply:
I was wrong. I have now updated this to 100%